src/components/ButtonFloat/ButtonFloat.stories.tsx

```diff
@@ -18,66 +17,77 @@ export default {
   },
   decorators: [
     (story) => (
-      <div
-        style={{
-          position: 'fixed',
-          display: 'flex',
-          justifyContent: 'center',
-          alignItems: 'center',
-          top: 0,
-          right: 0,
-          bottom: 0,
-          left: 0,
-        }}
-      >
-        {story()}
+      <div style={{ width: '100%', height: '50px' }}>
```

Comment:
is this the fix to avoid the button float to be cut by other box?

Reply:
Yep!